bedrock/privacy/redirects.py

```diff
@@ -0,0 +1,9 @@
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (
+    # bug 1319207
+    # 'Firefox Focus' cannot be used in de locale due to legal constraints
+    redirect(r'^de/privacy/firefox-focus/?', '/de/privacy/firefox-klar/', locale_prefix=False),
+    # special de URL should not be accessible from other locales
+    redirect(r'^(?P<locale>(?!de)\w{2,3}(?:-\w{2})?/)privacy/firefox-klar/?', 'privacy.notices.firefox-focus', locale_prefix=False),
```

Comment:
instead of this complex regex you could use the `no_redirect` helper which is also in the `bedrock.redirects.util` lib. It stops redirect processing and forces the normal Django process. So this could be:

```python
from bedrock.redirects.util import no_redirect, redirect

redirectpatterns = (
    # bug 1319207
    # 'Firefox Focus' cannot be used in de locale due to legal constraints
    redirect(r'^de/privacy/firefox-focus/?', '/de/privacy/firefox-klar/', locale_prefix=False),
    # special de URL should not be accessible from other locales
    no_redirect(r'^de/privacy/firefox-klar/?', locale_prefix=False),
    redirect(r'^privacy/firefox-klar/?', 'privacy.notices.firefox-focus'),
)
```

Both of these work, but I find the one with `no_redirect` a bit more readable. Just a suggestion though. I'm happy with what you have since it works :)